go.mod

```diff
@@ -175,6 +175,8 @@ replace (
 
 replace github.com/telepresenceio/telepresence/rpc/v2 => ./rpc
 
+replace github.com/datawire/ambassador v1.13.7-0.20210527054604-663dfb393e59 => github.com/donnyyung/emissary v1.14.2-0.20210909174709-51a94441b88c
```

Comment:
So I believe this will actually go away once https://github.com/telepresenceio/telepresence/pull/2093 is merged since it's included to get the kates updates I was trying to add.  If it for some reason doesn't, I will add the comment.